### ISSUE TITLE
Input field style inconsistent with bulma input style

### ISSUE THREAD

**Issue by svkurowski** [Wikiki/bulma-calendar#171]
## Bug Report

**Environment**
- Extension version(s): [From demo](https://creativebulma.net/product/calendar/demo)
- Bulma version: [From documentation](https://bulma.io/documentation/form/input/)
- OS: macOS 10.14.4 
- Browser: Chrome 73, didn't test others
- How you are customizing the extension: no

**Current Behavior**
This is what a text input field looks like in Bulma:
![image](https://user-images.githubusercontent.com/2794645/56446160-509b8f80-6301-11e9-931c-d7db77e62cdd.png)

And this is what the date input text field looks like in Bulma-Calendar (screenshot from demo):
![image](https://user-images.githubusercontent.com/2794645/56446180-6c9f3100-6301-11e9-99e7-d46475c8a266.png)
The border is not rounded, the drop shadows are not there, hover and focus border colors are different/not there.

**Comment by victorbueno** [comment]
+1

**Comment by YoanaDimitrova-kpmg** [comment]
+1

**Comment by odinsride** [comment]
Hello, I had this problem as well and overrode the classes in a rails app to somewhat match the Bulma styling.  It's not perfect and doesn't account for hover/focus since I don't particularly need that, but hopefully this will help get you started.  The extension itself is very nice, just a shame it doesn't follow Bulma styling!

```css
// Import Bulma Core
@import "./bulma-0.7.5/bulma.sass";

// Import Bulma Extensions
@import "./bulma-0.7.5/sass/extensions/_all";

// Override datetimepicker z-index
.datetimepicker {
    z-index: 99;
}

.datetimepicker-dummy {
    .datetimepicker-dummy-wrapper {
        border: 1px solid $primary;
        border-radius: 4px;
        box-shadow: inset 0 1px 2px rgba(10, 10, 10, 0.1)
    }
}```

**Comment by jasper-moment** [comment]
+1

**Comment by michael-hack** [comment]
Fixed in 6.1.10
